functionaltest/testng.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="BDD Test Suite" verbose="1" parallel="tests" thread-count="1" configfailurepolicy="continue">
+    <test name="Login and Update Profile Test" annotations="JDK" preserve-order="true">
+        <classes>
+            <class name="TestRunner"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
Will the test work without this file?

Reply:
I think so, but let me try without it. I thought it's good to leave it here in case to want to add more functionalities to the functional tests.